api/types/transfer/streaming.proto

```diff
@@ -0,0 +1,29 @@
+/*
+	Copyright The containerd Authors.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+syntax = "proto3";
+
+package containerd.v1.types;
+
+option go_package = "github.com/containerd/containerd/api/types/transfer";
+
+message Data {
```

Comment:
Are these the messages that will be used by api/streaming service?

Reply:
I need to document this a little more, the streaming API is only responsible for managing and identifying the stream. The protocol of the stream, including the messages sent across the stream are determined by APIs which use them. Inside the API messages, there is just a stream ID string communicated, but the context of where it is used determines the protocol. At the very least the protocol should be defined or mentioned next to where the stream ID message is in the proto files.